src/Avalonia.Controls/ProgressBar.cs

```diff
@@ -20,17 +20,47 @@ public class ProgressBar : RangeBase
         public static readonly StyledProperty<Orientation> OrientationProperty =
             AvaloniaProperty.Register<ProgressBar, Orientation>(nameof(Orientation), Orientation.Horizontal);
 
-        private static readonly DirectProperty<ProgressBar, double> IndeterminateStartingOffsetProperty =
+        public static readonly DirectProperty<ProgressBar, double> ContainerAnimationStartPositionProperty =
```

Comment:
Those new props are from UWP's [ProgressBarTemplateSettings](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.primitives.progressbartemplatesettings?view=winrt-19041), separated from ProgressBar class with point to hide template properties. Also they are get-only.

As for me It's fine to move them to ProgressBar.cs class, but should we?

Reply:
Sorry im not familiar in that part of UWP, but idk what's the point of that primitive though? If isnt gonna be used by anything else other than ProgressBar then i dont think it needs to be split from it..

Reply:
I think it is just to move template specific props out. It's common pattern for some controls in UWP.